base/deprecated.jl

```diff
@@ -313,4 +313,17 @@ const var"@_noinline_meta" = var"@noinline"
 
 @deprecate splat(x) Splat(x) false
 
+# We'd generally like to avoid direct external access to internal fields
+# Core.Compiler.inlineable and Core.Compiler.set_inlineable! move towards this direction,
+# but we need to keep these around for compat
+@noinline function getproperty(ci::CodeInfo, s::Symbol)
```

Comment:
I think this `@noinline` can potentially impact the performance of our compiler. This operation is hot.